tests/unit/literaturesuggest/test_literaturesuggest_form.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+from inspirehep.modules.literaturesuggest.forms import LiteratureForm
+
+
+def test_degree_choices_contains_capitalized_phd():
+    """Ensure that lowercased degree type in the schema ends up with the
+    right capitalization in the form.
+    """
+    literature_form = LiteratureForm()
+
+    assert ('phd', 'PhD') in literature_form.degree_type.choices
```

Comment:
But how this happens is an implementation detail, which is not a thing worth testing. If we change how the form library internally stores its data, this test is going to fail, even if everything in the UI looks correct.

BTW, this was not even what the user wanted! They wanted something solved by https://github.com/inspirehep/inspire-next/pull/2211.